CHANGELOG.md

```diff
@@ -32,11 +32,10 @@ For more information, see [UP42 Python package description](https://pypi.org/pro
 
 ## 0.33.0
 
-**October XX, 2023**
+**November 14, 2023**
 
-A new function added to the [Asset class](https://sdk.up42.com/reference/asset-reference):
-
-- `get_stac_asset_url` generates a signed URL that allows you to download a STAC asset from storage without authentication.
+- Updated [authentication](https://sdk.up42.com/authentication) by changing it from project-based to account-based.
+- Added a new function added to the [Asset class](https://sdk.up42.com/reference/asset-reference): `get_stac_asset_url` generates a signed URL that allows you to download a STAC asset from storage without authentication.
```

Comment:
This is missing in the documentation

Reply:
And I don't see the change in the PR as well

Reply:
`you` is not needed

Reply:
I don't understand what you mean by "the change", What change do you mean that is not in the PR? Can you be more specific.

Reply:
In addition, the changelog is pointing to the authentication docs, can you be more specific on what is missing in the documentation? This is a reference to the authentication docs.

Reply:
The change - I mean the line L38 I commented on about get_stac_asset_url

Reply:
OK, this change was an edition to the previous changelog prepared for a smaller PR related to the signed URL feature of the release 0.33.0. you can see the changes on this commit https://github.com/up42/up42-py/commit/5cc799f470a9e0cb99a30aeab46e00c087326896. Please take into account that automatic release is still not fully enabled.